### ISSUE TITLE
Bug: quirky handling of string outputs

### ISSUE THREAD

**Issue by getify** [lukehaas/RunJS#55]
<img width="846" alt="screen shot 2019-02-27 at 10 21 37 pm" src="https://user-images.githubusercontent.com/150330/53523812-15899680-3ade-11e9-923e-1467ca36c93d.png">

Output lines 3 and 4 seem buggy here, no? Line 4 is especially surprising given how it differs from line 2.

I would have expected those output lines to say:

```js
"Hello "world"!"    // or:   'Hello "world"!'
42 "Hello 'world'!"
```

**Comment by lukehaas** [comment]
I've made an update that resolves the inconsistency with the 4th output. 
I've also to switched to using single quotes as it seems more consistent with other outputs.
As a result your example will now look like this in the next release:

<img width="610" alt="screenshot 2019-02-28 at 21 23 53" src="https://user-images.githubusercontent.com/741344/53599719-bee49100-3b9f-11e9-84f9-9b618fc0632a.png">

What do you think?

**Comment by getify** [comment]
OK I can live with that. :) Thanks!

**Comment by lukehaas** [comment]
This is now resolved in version 1.3.1
